test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -272,6 +273,32 @@ define([
             });
         });
 
+        it('Should start up in "editingComplete" state when initialized with proper data', async () => {
+            const cell = Mocks.buildMockCell('code');
+            Jupyter.notebook = Mocks.buildMockNotebook();
+            const cellWidget = BulkImportCell.make({
```

Comment:
do you need the `cellWidget` variable or can you just run `BulkImportCell.make({..})`?

Reply:
Nope, I was experimenting with deleting the cell (and stopping widgets and such) at the end. That, however, probably means removing / canceling running Promises, and making sure that the `make` function returns as a Promise anyway. I just forgot to undo that part.